components/Datasets/DescribeDataset.vue

```diff
@@ -288,6 +294,57 @@ Elle aide les utilisateurs à comprendre rapidement ce qu’il contient et amél
             >
               {{ getFirstWarning("description_short") }}
             </SimpleBanner>
+            <div class="flex items-center gap-4 mt-2 mb-3">
+              <Tooltip v-if="!canGenerateDescriptionShort">
+                <BrandedButton
+                  type="button"
+                  color="primary"
+                  :disabled="true"
+                >
+                  <div class="flex items-center space-x-2">
+                    <NuxtImg
+                      src="/illustrations/sparkle.svg"
+                      alt=""
+                      class="size-4"
+                    />
+                    <span>{{ $t('Suggérer une description courte') }}</span>
+                  </div>
+                </BrandedButton>
+                <template #tooltip>
+                  {{ $t('Remplissez le titre et une description d\'au moins {min} caractères pour utiliser cette fonctionnalité.', { min: DESCRIPTION_MIN_LENGTH }) }}
+                </template>
+              </Tooltip>
+              <BrandedButton
+                v-else
+                type="button"
+                color="primary"
+                :disabled="isGeneratingDescriptionShort"
+                @click="handleAutoCompleteDescriptionShort"
+              >
+                <div class="flex items-center space-x-2">
+                  <NuxtImg
+                    v-if="!isGeneratingDescriptionShort"
+                    src="/illustrations/sparkle.svg"
```

Comment:
Can you use `RiSparklingLine` and `RiSparklingFill` components from remixicon instead of svg ?